Guide-to-migrate-e2e-tests.md

```diff
@@ -0,0 +1,65 @@
+## Overview
+
+Currently, Oppia E2E tests are in a hybrid state where we have both WebdriverIO and Protractor test suites. The project **Migrate away from Protractor** aims to migrate all the existing E2E tests to WebdriverIO.
+
+This guide is to help contributors write/modify E2E tests in webdriverIO during the hybrid state.
+
+## Selectors
+
+The `$` command is a short way to call the findElement command in order to fetch a single element on the page
+
+| Name                                     | Protractor                               |WebdriverIO           |
+| -----------------------------------------| -----------------------------------------|----------------------|
+| By class name                            |`element(by.css('.className'))`           |`$('.className')`     |
+| Element containing certain string Text   |`element(by.cssContainingText(tag, text))`|`$('tag=text')`       |
+| Multiple Elements                        |`element.all(by.css('.className'))`       |`$$('.className')`    |
+| By ID                                    |`element(by.css('#idName'))`              |`$('#idName')`        |
+| Chain Selectors                          |`element(by.css('some-css')).element(by.css('tag-within-css'))`  | `$('some-css').$('tag-within-css’)`      |
```

Comment:
"Chain Selectors" isn't really a name of a selector type. Maybe the first column should instead describe what goal the selector accomplishes

Reply:
Done.